spec/modules/helpers.rb

```diff
@@ -71,4 +83,85 @@ def delete_category(category_name, category_description)
     expect(page).to have_text("Category was successfully destroyed.")
     expect(page).not_to have_selector("#category_#{category_id}")
   end
+end
+
+module DiscussionHelpers
+  include SharedMatchers
+  include DiscussionMatchers
+
+  def create_discussion(discussion_title, discussion_body, category_name = DEFAULT_CATEGORY_NAME)
+    create_category(category_name, CATEGORY_DESCRIPTION)
```

Comment:
This is using the constant.  However, I'm wondering if this should even be present.  As written, this will manually test the create category workflow.  This adds time and doesn't need to happen. For category system tests that are testing the category creation process, it should do that.  For tests on discussions, a category should be created and passed in if necessary.